### PR TITLE
Version 4.4

### PR DESCRIPTION
**Pull request by ramijebara**
Please review these changes.

### CODE REVIEW

File: documentation/release-notes/topics/version_4.rst [hunk 1]
```diff
@@ -2,10 +2,61 @@
...
+
+
+===============================
+New Feature - Business Entities
+===============================
+Business Entities allow customers to track customers, partners, and department associations to assets, connections and circuits. Business Entities can have contacts and addresses as well as:
+
+- Access Control
+- Change Log
+- Custom properties
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:23]: For consistency, this should be "Custom Properties" or they should all be formatted like this.

File: documentation/release-notes/topics/version_4.rst [hunk 2]
```diff
@@ -2,10 +2,61 @@
...
+
+==================================
+Enhanced Feature: New Bulk Actions
+==================================
+The bulk actions system has been updated to allow Business Entities to be associated in bulk with assets.
+
+===============================================
+Enhanced Feature: Connection and Circuit import
+===============================================
+The Connections and Circuits import system has been updated to allow for updating and setting of business entities.
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:43]: "business entities" is formatted as "Business Entities" everywhere else.

File: documentation/release-notes/topics/version_4.rst (hunk 1 above)
Comment (ramijebara) [reply documentation/release-notes/topics/version_4.rst]: Fixed! Thank you.

File: documentation/release-notes/topics/version_4.rst (hunk 2 above)
Comment (ramijebara) [reply documentation/release-notes/topics/version_4.rst]: Fixed! Thank you.

**Review by kyledickau — APPROVED** [review]

**Pull request merged**